charts/nri-bundle/values.yaml

```diff
@@ -1,106 +1,129 @@
-
-# It is possible to configure dependencies values from this chart.
-# For example you can configure `verboseLog` variable of the newrelic-infrastructure chart installed as a dependency and enable openshift configs as follows.
-
-# newrelic-infrastructure:
-#   verboseLog: true
-#   openshift:
-#     enabled: true
-#     version: "4.x"
-
-# The same approach can be followed to update any of the dependencies.
-# https://helm.sh/docs/chart_template_guide/subcharts_and_globals
-
-# If you wish to monitor services running on Kubernetes you can provide integrations
-# configuration under integrations_config that is pass down to the newrelic-infrastructure chart. You just need to create a new entry where
-# the "name" is the filename of the configuration file and the data is the content of
-# the integration configuration. The name must end in ".yaml" as this will be the
-# filename generated and the Infrastructure agent only looks for YAML files. The data
-# part is the actual integration configuration as described in the spec here:
-# https://docs.newrelic.com/docs/integrations/integrations-sdk/file-specifications/integration-configuration-file-specifications-agent-v180
-
-# For example, if you wanted do to monitor a Redis instance that has a label "app=redis"
-# you could do so by adding following entry:
-#
-# newrelic-infrastructure:
-#   integrations_config:
-#     - name: nri-redis.yaml
-#       data:
-#         discovery:
-#           command:
-#             # Run NRI Discovery for Kubernetes
-#             # https://github.com/newrelic/nri-discovery-kubernetes
-#             exec: /var/db/newrelic-infra/nri-discovery-kubernetes
-#             match:
-#               label.app: redis
-#         integrations:
-#           - name: nri-redis
-#             env:
-#               # using the discovered IP as the hostname address
-#               HOSTNAME: ${discovery.ip}
-#               PORT: 6379
-#             labels:
-#               env: test
+
+# -- change the behaviour globally to all the supported helm charts.
+# See [user's guide of the common library](https://github.com/newrelic/helm-charts/blob/master/library/common-library/README.md) for further information.
+# @default -- See [`values.yaml`](values.yaml)
+global:
+  # -- The cluster name for the Kubernetes cluster.
+  cluster: ""
+
+  # -- The license key for your New Relic Account. This will be preferred configuration option if both `licenseKey` and `customSecret` are specified.
+  licenseKey: ""
+  # -- The license key for your New Relic Account. This will be preferred configuration option if both `insightsKey` and `customSecret` are specified.
+  insightsKey: ""
+  # -- Name of the Secret object where the license key is stored
+  customSecretName: ""
+  # -- Key in the Secret object where the license key is stored
+  customSecretLicenseKey: ""
+  # -- Key in the Secret object where the insights key is stored
+  customSecretInsightsKey: ""
+  # -- Additional labels for chart pods
+
```

Comment:
needed extra line?

Reply:
No, when I was moving values up and down I made the line jump in the incorrect line.

In fact, that issue made `podLabels` not to template in the README.

Solved in [#4f2332e](https://github.com/newrelic/helm-charts/pull/795/commits/4f2332e6fc57c5e9842ddb8e958681644a537814)